mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,16 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(isHappy) {
+  if (isHappy === true) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
-function greaterThan10(num) {
-  let isBigEnough;
-
-  if (isBigEnough) {
+function greaterThan10(isBigEnough) {
```

Comment:
hi 
here is some sort of discrepancy
whether it's a variable name or function's argument `isBigEnough` it supposed to have Boolean value.
please check this link out: [Tips on naming boolean variables - Cleaner Code](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)  
So, argument `num` was actually good. It stood for "Number".

But, what could be `let isBigEnough;` ? 
hint - this is result of comparison that always returns some Boolean value: true or false.